README.md

```diff
@@ -252,18 +252,17 @@ You can provide additional configuration options using the `-DVARIABLE=VALUE` sy
 Here are all the options that are supported:
 * `WITH_OPENSSL` : Set to `ON` to build the library with SSL support.
 This will require [OpenSSL](https://www.openssl.org) to be installed on your system. The default is `OFF`.
-* `BUILD_STATIC_LIB` : Set to `ON` or `OFF` depending on whether you want the static library. The default is `OFF`.
-* `BUILD_SHARED_LIB` : Set to `ON` or `OFF` depending on whether you want the shared library. The default is `ON`.
+* `BUILD_SHARED_LIBS` : Set to `ON` or `OFF` depending on whether you want the shared(ON) or static(OFF) library. The default is `ON`.
 * `DISABLE_LOGGING` : Setting this option to `ON` disables logging. The default is `OFF`.
 
 ##### 1.1.5.2.1 Example Configuration Commands
 Build only the static library with SSL support:
 ```sh
-cmake .. -DWITH_OPENSSL=ON -DBUILD_SHARED_LIB=OFF -DBUILD_STATIC_LIB=ON
+cmake .. -DWITH_OPENSSL=ON -DBUILD_SHARED_LIBS=OFF
 ```
-Build both the shared and static library without SSL support:
+Build both the shared library without SSL support:
```

Comment:
I think we can delete this example, because it does the same with the default `cmake ..` (SSL is off by default).
The above example is explanatory enough.

You may also consider removing the section header "1.1.5.2.1" completely, and put the above example under "1.1.5.2".

Reply:
Deleted the line. Removed the section "1.1.5.2.1".